src/controllers/controller.line.js

```diff
@@ -248,30 +249,44 @@ module.exports = function(Chart) {
 			var meta = me.getMeta();
 			var area = me.chart.chartArea;
 
-			// only consider points that are drawn in case the spanGaps option is ued
+			// Only consider points that are drawn in case the spanGaps option is used
 			var points = (meta.data || []).filter(function(pt) { return !pt._model.skip; });
 			var i, ilen, point, model, controlPoints;
 
-			var needToCap = me.chart.options.elements.line.capBezierPoints;
```

Comment:
Right, I missed that the `capControlPoint` was behind the setting. 

You are right that it will probably be a bit faster :)